tf2_ros/include/tf2_ros/transform_listener.h

```diff
@@ -133,13 +133,12 @@ class TransformListener
       // Duplicate to modify option of subscription
       rclcpp::SubscriptionOptionsWithAllocator<AllocatorT> tf_options = options;
```

Comment:
This temporary copy can be removed too.

Reply:
all comments are addressed in https://github.com/ros2/geometry2/pull/492/commits/5b4a1844302861d5c6929ef2600f0127721985d8.

Note: I put you as `Co-authored-by` since I borrowed your suggested code.